bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -341,6 +341,11 @@ default String getPrefillService() {
         return null;
     }
 
+    @JsonIgnore
+    default String getRoleAttribute() {
```

Comment:
This should be part of Container interface, since all containers can have role attributes

Reply:
Moved to Interface!